src/App.tsx

```diff
@@ -11,6 +14,9 @@ const theme = createTheme();
  * This prevents git diff churning
  */
 const providers = [
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  <ApolloProvider client={apolloClient} />,
```

Comment:
Is it complaining about missing children? If so you can do the same thing `ThemeProvider` is doing right below.

Reply:
Didn't even catch the `ThemeProvider` line, doh!